packages/grafana-ui/src/components/FileUpload/FileUpload.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, FormEvent, useState } from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { css, cx } from 'emotion';
+import { getFormStyles, Icon } from '../index';
+import { stylesFactory, useTheme } from '../../themes';
+
+export interface Props {
+  onFileUpload: (event: FormEvent<HTMLInputElement>) => void;
+  /** Custom upload label text */
+  label?: string;
```

Comment:
Since this is button-ish component can we use `children` instead of `label` to keep the API similar?:

```
<FileUpload ...>
  Upload file...
</FileUpload>
```